data/src/main/kotlin/com/vultisig/wallet/data/api/models/SolanaJson.kt

```diff
@@ -53,7 +54,8 @@ data class SolanaFeeObjectRespJson(
 @Serializable
 data class BroadcastTransactionRespJson(
     @SerialName("error")
-    val error: String?,
+    @Serializable
+    val error: JsonObject?,
```

Comment:
The purpose is to prevent exceptions due to deserializing the response when the error is not empty.